plugins/foreman_openscap/0.4/index.md

```diff
@@ -0,0 +1,187 @@
+
+
+## 1.2 Release Notes
+OpenSCAP has two versions: 0.3.4 for Foreman 1.7 and 0.4.x for 1.8+.
+
+Both versions' functionality is the same. The changes reflect the changes in communicating with Foreman API and Smart-Proxy.
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Foreman version</th>
+    <th>Plugin version</th>
+    <th>Proxy version</th>
+    <th>Client version</th>
+  </tr>
+  <tr>
+    <td>= 1.7</td>
+    <td>0.3.4</td>
+    <td>0.3.1</td>
+    <td>0.1.1</td>
+  </tr>
+  <tr>
+    <td>>= 1.8</td>
+    <td>0.4.1</td>
+    <td>0.4.0</td>
+    <td>0.1.1</td>
+  </tr>
+</table>
+
+# 2. Installation
+
+There are a few components to install: 
+
+Foreman OpenSCAP plugin, Foreman Proxy OpenSCAP plugin (aka Smart Proxy OpenSCAP), foreman_scap_client and puppet-foreman_scap_client.
+
+### 2.1 Installing foreman_openscap
+
+Please refer to Foreman install plugin [manual](http://theforeman.org/manuals/1.8/index.html#6.1InstallaPlugin) to install the plugin on Foreman.
+
+### 2.2 Installing smart_proxy_openscap
+
+On the Smart-Proxy server:
+
+    
+    > gem install smart_proxy_openscap -v '<version>'
+    > cp /path/to/gem/smart_proxy_openscap-<version>/settings.d/openscap.yml.example /path/to/foreman-proxy/settings.d/openscap.yml
+    
+Edit ```openscap.yml``` with the appropriate settings
+
+    ---
+    :enabled: true
+
+
+### 2.3 Installing puppet-foreman_scap_client
+
+    puppet module install isimluk-foreman_scap_client
+    
+Running puppet on the client host, will install also foreman_scap_client.
+
+
+# 4. Usage 
+
+This chapter covers features that you can use in terms of Foreman and OpenSCAP
+integration. Everything described below assumes you've sucessfully installed 
+foreman_openscap, smart_proxy_openscap and puppet-foreman_scap_client is available on your puppetmaster and Foreman.
+
+You would usually start with uploading SCAP contents, then create policies of those SCAP contents and assign the policy to hosts or hostgroups.
+The puppet module will install ```foreman_scap_client``` and configure it with the needed policy information. The puppet module also adds a cron line,
+which runs the scap client at the schedule select when creating the policy.
+
+## 4.1 Creating SCAP content
+
+foreman_openscap comes with default scap content provided by [scap-security-guide](https://fedorahosted.org/scap-security-guide/).
+
+The default SCAP content is available for RHEL 6, RHEL 7 and Fedora.
+
+Besides the default SCAP content, you can also upload your own SCAP content.
+
+__Access SCAP contents__ Hosts -> Compliance -> SCAP Contents
+
+__Create SCAP Content__ - You can upload any valid OpenSCAP DataStream file
+![New SCAP Content]({{page.images}}/create_new_scap_content.png)
+
+
+## 4.2 Creating Policy wizard
+
+* First step is to name your policy. 
```

Comment:
No need for connectors if you're going to use bullet points.

* Name your policy.
* Choose which SCAP content & SCAP profile to apply.
* etc...